types/hooks/screens/useChatUtilsHook.types.ts

```diff
@@ -13,6 +14,7 @@
   goToChatInfoScreen: () => void;
   goBackFromChatScreen: () => void;
   handleTextSystem: (item: any) => string;
+  setSelectedChannel: (channel: ChannelList) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setSelectedChannel` function in the interface is also a good move. It's always a good practice to define all the functions that a hook should implement in its interface. Similar to the previous comment, consider adding a comment to describe the purpose of this function.

```typescript
// Sets the selected channel
setSelectedChannel: (channel: ChannelList) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->